src/assets/scss/components/_videoComponent.scss

```diff
@@ -39,7 +40,7 @@
 /* Background Header */
 .img-bg-header {
 	background: linear-gradient(rgba(0, 0, 0, 0.3), rgba(0, 0, 0, 0.3)), url("../../../public/images/bg-header.jpg") center center;
```

Comment:
Since it's in the file _videoComponent.scss it is compiled by the loader. The relative path is the way to go. Otherwise, it doesn't work. 
Ex: 
1)  ./images/bg-header.jpg  --> doesn't work
2)  ../../../public/images/bg-header.jpg --> works